README.md

```diff
@@ -0,0 +1,950 @@
+[Comparisons]: #comparisons
+
+| Operator | Argument(s)                    | Example                          |
+|----------|--------------------------------|----------------------------------|
+| `==`     | `Selector, IPLD`               | `["==", ".a", [1, 2, {"b": 3}]]` |
+| `<`      | `Selector, (integer \| float)` | `["<",  ".a", 1]`                |
+| `<=`     | `Selector, (integer \| float)` | `["<=", ".a", 1]`                |
+| `>`      | `Selector, (integer \| float)` | `[">",  ".a", 1]`                |
+| `>=`     | `Selector, (integer \| float)` | `[">=", ".a", 1]`                |
+
+Literal equality (`==`) MUST match the resolved selecor to entire IPLD argument. This is a "deep comparison".
+
+Numeric inequalities MUST be agnostic to numeric type. In other words, the decimal representation is considered equivalent to an integer (`1 == 1.0 == 1.00`). Attempting to compare a non-numeric type MUST return false and MUST NOT throw an exception.
+
+## Glob Matching
+[Glob Matching]: #glob-matching
+
+| Operator | Argument(s)         | Example                               |
+|----------|---------------------|---------------------------------------|
+| `like`   | `Selector, Pattern` | `["like", ".email", "*@example.com"]` |
+
+Glob patterns MUST only include one specicial character: `*` ("wildcard"). There is no single character matcher. As many `*`s as desired MAY be used. Non-wildcard `*`-literals MUST be escaped (`"\*"`). Attempting to match on a non-string MUST return false and MUST NOT throw an exception.
+
+The wildcard represents zero-or-more characters. The following string literals MUST pass validation for the pattern `"Alice\*, Bob*, Carol.`:
+
+* `"Alice*, Bob, Carol."`
+* `"Alice*, Bob, Dan, Erin, Carol."`
+* `"Alice*, Bob  , Carol."`
+* `"Alice*, Bob*, Carol."`
+
+The following MUST NOT pass validation for that same pattern:
+
+* `"Alice*, Bob, Carol"` (missing the final `.`)
+* `"Alice*, Bob*, Carol!"` (final `.` MUST NOT be treated as a wildcard)
+* `"Alice, Bob, Carol."` (missing the `*` after `Alice`)
+* `"Alice Cooper, Bob, Carol."` (the `*` after `Alice` is an escaped literal in the pattern)
+* `" Alice*, Bob, Carol. "` (whitespace in the pattern is significant)
+
+## Connectives
+[Connectives]: #connectives
+
+Connectives add context to their enclosed statement(s).
+
+| Operator | Argument(s)   | Example                                    |
+|----------|---------------|--------------------------------------------|
+| `and`    | `[Statement]` | `["and", [[">", ".a", 1], [">", ".b", 2]]` |
+| `or`     | `[Statement]` | `["or",  [[">", ".a", 1], [">", ".b", 2]]` | 
+| `not`    | `Statement`   | `["not", [">", ".a", 1]]`                  |
+
+### And
+[And]: #and
+
+`and` MUST take an arbitrarily long array of statements, and require that every inner statement be true. An empty array MUST be treated as true.
+
+```js
+// Data
+{ name: "Katie", age: 35, nationalities: ["Canadian", "South African"] }
+
+["and", []]
```

Comment:
If I'm not mistaken, all those policy example should be wrapped with `[ <example goes here> ]`, otherwise there are not legal as per the syntax

Reply:
I suppose those are statements, not policies, but that's a bit confusing.